ocaml/libs/tracing/tracing.ml

```diff
@@ -11,6 +11,25 @@
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU Lesser General Public License for more details.
  *)
+
+module D = Debug.Make (struct let name = "tracing" end)
+
+open D
+
+type endpoint = Bugtool | Url of string [@@deriving rpcty]
```

Comment:
Why is the Url a `string` instead of a`Uri.t`? If this is because Uri.t would lack a deriving I have some code that could help with that.

Reply:
It makes it easy to derive yes, and also to quite easy to get from a string. I am not aware of `Uri.t` so I will look into that.

Reply:
@snwoods

Reply:
Resolved by aab0e0a8f3e772eb2a12232be92100c0cfac94dc